components/Home/Bonding/TokenManagement/hooks/useWhirlpool.jsx

```diff
@@ -9,15 +9,20 @@ import { BN } from '@coral-xyz/anchor';
 import { gql, GraphQLClient } from 'graphql-request';
 import { areAddressesEqual } from '@autonolas/frontend-library';
 
-import { ADDRESSES } from 'common-util/Contracts';
+import { ADDRESSES } from 'common-util/constants/addresses';
 import { useSvmConnectivity } from 'common-util/hooks/useSvmConnectivity';
 import { getCalculatedPriceLp } from '../../BondingList/utils';
 import { WHIRLPOOL, ORCA } from '../constants';
 
 const whirlpoolQuery = async () => {
   const SHYFT_API_KEY = process.env.NEXT_PUBLIC_SHYFT_API_KEY;
   if (!SHYFT_API_KEY) {
-    throw new Error('SHYFT_API_KEY is not available');
+    if (process.env.NODE_ENV === 'development') {
+      console.error('SHYFT_API_KEY is not available');
```

Comment:
SHYFT doesn't seem to work on development localhost
i removed thrown error and just return an empty object instead

Reply:
I'm using the one from Vercel prod env atm, perhaps it's limited by origin?
I get CORS errors when running from local